packages/client/src/views/Dashboard.vue

```diff
@@ -104,7 +104,7 @@
     </b-row>
     <br v-if="!showStats" />
 
-    <b-card title="Total Interested Grants by Agencies">
+    <b-card title="Total Interested Grants by Agencies" v-if="showStats" >
```

Comment:
Does hiding this here also eliminate making a request to the backend on page load?

Reply:
@jakekreider I updated the code a bit to ensure we do not load this data from the backend as well.

Reply:
Good catch, thanks! And I appreciate the screen shots as well.